.github/workflows/main.yml

```diff
@@ -20,13 +17,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       
-      - name: Install & Build # 執行指令
+      - name: Install & Build
         run: |
           npm i -g pnpm
           pnpm install
           pnpm run build 
           
-      # Runs a single command using the runners shell
       - name: Deploy
         uses: JamesIves/github-pages-deploy-action@releases/v4
         with:
```

Comment:
The code patch looks fine, but here are a few suggestions:

- Instead of `name: ci`, consider using a more descriptive name that reflects the purpose of the workflow.
- Consider adding a `pull_request` event trigger, so that pull requests can also be tested before merging into the main branch.
- Add error handling steps to catch any errors that may arise during the build or deployment process.
- Consider using specific versions for external dependencies, instead of relying on default/latest versions from the package manager. 
- Lastly, consider reviewing whether it's necessary to install and use an npm package globally (`npm i -g pnpm`). It may be better to install packages locally in the project directory by adjusting the path or environment variables (e.g., `./node_modules/.bin/pnpm` or `npx pnpm`).